reports_api/reports/schema.py

```diff
@@ -696,6 +722,55 @@ def resolve_paid_tickets(self, info, summitId=0):
             return 'Yes' if attendee.tickets.filter(status='Paid', is_active=True).count() > 0 else 'No'
 
         return 'No'
+
+    def resolve_selection_plan(self, info, summitId):
+        selection_plan_names = []
+        seen_ids = set()
+        presentations = self.presentations.filter(summit_id=summitId)
+        for presentation in presentations:
+            if not presentation.selection_plan is None and presentation.selection_plan_id not in seen_ids:
+                seen_ids.add(presentation.selection_plan_id)
+                selection_plan_names.append(presentation.selection_plan.name)
+
+        return ' || '.join(x for x in selection_plan_names)
+
+    def resolve_submission_status(self, info, summitId):
+        results = []
+        seen_ids_and_statuses = set()
+        presentations = self.presentations.filter(summit_id=summitId)
+        for presentation in presentations:
+            id_status_pair = (presentation.id, presentation.submission_status)
+            if id_status_pair not in seen_ids_and_statuses:
+                seen_ids_and_statuses.add(id_status_pair)
+                results.append(SubmissionStatusType(presentation.id,
+                                          presentation.submission_status,
+                                          presentation.category.id,
+                                          presentation.category.title))
+
+        return results
+
+    def resolve_selection_status(self, info, summitId, **kwargs):
+        results = []
+        seen_ids_and_statuses = set()
+        presentations = self.presentations.filter(summit_id=summitId)
+        selected_presentations = SelectedPresentation.objects.filter(presentation_id__in=presentations)
```

Comment:
this is not needed
u need to iterate self.presentations.filter(summit_id=summitId) and return the selection_status of the presentation see  https://github.com/fntechgit/summit-reports-api/pull/13/files#r1718791427